modules/rosa-getting-started-revoke-admin-privileges.adoc

```diff
@@ -0,0 +1,73 @@
+// Module included in the following assemblies:
+//
+// * rosa_getting_started/rosa-getting-started.adoc
+
+[id="rosa-getting-started-revoke-admin-privileges_{context}"]
+= Revoking administrator privileges for a user
+
+Follow the steps in this section to revoke `cluster-admin` or `dedicated-admin` privileges for a user.
```

Comment:
Consider "...a user's `cluster...`" or "...from a user."

Reply:
Thanks for the suggestion. I have updated this to "from a user", to avoid using a possessive.